measures/measures-schema.yaml

```diff
@@ -28,6 +28,8 @@ definitions:
         description: ACI measures can belong to the transition measure set. Quality measures can belong to multiple measure sets that represent different specialties.
         type: array
         items: { $ref: #/definitions/measureSets }
+      benchmark:
```

Comment:
I may have missed a follow up conversation with Mari but I'm not sure we want to store benchmarks nested in measures since measures can be multi year but benchmarks can only represent a single year.